cert-manager/controllers/keptnwebhookcontroller/keptnwebhookcertificate_controller.go

```diff
@@ -68,10 +64,46 @@ func (r *KeptnWebhookCertificateReconciler) Reconcile(ctx context.Context, reque
 
 	r.ctx = ctx
 
-	_, err := r.getMutatingWebhookConfiguration()
+	mutatingWebhookConfiguration, err := r.getMutatingWebhookConfiguration()
```

Comment:
you could already pack all mutating webhook specific things into a separate parametrized func so that it will be easy in the future to add the same thing for conversion webhooks and others

Reply:
I added the logic for conversion webhook and this will be a bit different, however for the Validating webhook it will be a copy paste :)